### ISSUE TITLE
React Class is defined but never used

### ISSUE THREAD

**Issue by bartek** [feross/standard#81]
Hi there,

Love this project, but I'm curious if there's a way to handle this common React scenario?

```
var App = React.createClass({
   render: function () {
         return (
                 <div></div>
          )
    }
})

React.render(<App />, document.getElementById('application'))
```

App is obviously defined and being used, but I can see how the linter would be confused due to the JSX syntax. Any ideas?

Thanks!

**Comment by dcposch** [comment]
I ran into the same issue. I found a workaround: putting the React class in its own file and `require()`ing it

**Comment by dcposch** [comment]
I'll see if this still happens with just eslint, because jscs is going away soon!

**Comment by yoshuawuyts** [comment]
@dcposch `3.3.0` is out, you can check again

**Comment by justin808** [comment]
Is ReactJS, ES6, and JSX supported now?

**Comment by feross** [comment]
@justin808 Yes, they're supported now.

**Comment by justin808** [comment]
@feross Maybe I need the github tip? I got errors when converting these: https://github.com/justin808/react-webpack-rails-tutorial/tree/master/client/assets/javascripts

I'm considering converting my example of using Rails with React/Webpack/ES6.

**Comment by yoshuawuyts** [comment]
@justin808 Can you create a separate issue for this with your error messages included? Thanks!

**Comment by bartek** [comment]
Thanks @dcposch and co.!

**Comment by feross** [comment]
I think @dcposch's workaround might not be needed in the latest version of `standard` (3.3.1, just released) because of a bugfix in eslint 0.18.0: https://github.com/eslint/eslint/issues/2093

Hope that makes things smoother for you guys :)
